tests/sample-scenario-tests/src/test/java/org/wso2/SimpleScenarioTest.java

```diff
@@ -21,114 +21,114 @@
 
 /********** Test class for EI ***********/
 
-//import org.apache.commons.httpclient.*;
-//import org.apache.commons.httpclient.methods.GetMethod;
-//import org.testng.Assert;
-//import org.testng.annotations.Test;
-//
-//import java.io.IOException;
-//import java.util.concurrent.TimeUnit;
-//
-///**
-// * This class tests the HelloWorld Rest endpoint
-// */
-//
-//public class SimpleScenarioTest {
-//    @Test public void testEndpoint() throws InterruptedException {
-//        HttpClient client = new HttpClient();
-//        String deployedAPI = "/helloworld";
-//        int statusCode = -1;
-//        String uri = System.getProperty("endpoint") + deployedAPI;
-//        TimeUnit.MINUTES.sleep(1);
-//        HttpMethod method = new GetMethod(uri);
-//        try {
-//            statusCode = client.executeMethod(method);
-//        } catch (HttpException e) {
-//            System.err.println("Fatal protocol violation: " + e.getMessage());
-//            e.printStackTrace();
-//        } catch (IOException e){
-//            System.err.println("Fatal transport error: " + e.getMessage());
-//            e.printStackTrace();
-//        } finally {
-//            method.releaseConnection();
-//        }
-//        Assert.assertEquals(statusCode, HttpStatus.SC_OK);
-//    }
-//}
-
-
-/********** Test class for IS ***********/
-
-import org.apache.commons.httpclient.HttpStatus;
+import org.apache.commons.httpclient.*;
```

Comment:
Can we only import the required libraries, instead of importing all ?

Reply:
Fixed.